kong/plugins/moesif/handler.lua

```diff
@@ -62,6 +65,16 @@ function MoesifLogHandler:access(conf)
     res_body = res_body,
     req_post_args = req_post_args
   }
+
+  -- Check if need to block incoming request based on user-specified governance rules
+  local block_req = governance.govern_request(ngx, conf, start_access_phase_time)
+  if block_req == nil then 
+    if conf.debug then
+      ngx.log(ngx.DEBUG, '[moesif] No need to block incoming request.')
+    end
+    local end_access_phase_time = socket.gettime()*1000
+    ngx.log(ngx.DEBUG, "[moesif] access phase took time for non-blocking request - ".. tostring(end_access_phase_time - start_access_phase_time))
```

Comment:
is there a way to print time when it is blocked?

Reply:
We're doing it here - https://github.com/Moesif/kong-plugin-moesif/pull/33/files#diff-bf25e21931600691cf250b96dcae5bfdR204